tests/functional/test_layout.py

```diff
@@ -1,9 +1,8 @@
-from pangocffi import Context, FontDescription, Layout, Alignment, ffi
+from pangocffi import Context, FontDescription, Layout, Alignment, ffi, EllipsizeMode
```

Comment:
Looks good to me, except for a minor linting issue reported by flake8 which you can see in the automated CI checks here: https://travis-ci.org/github/leifgehrmann/pangocffi/jobs/727964075

I believe it can be fixed with just:

```suggestion
from pangocffi import Context, FontDescription, Layout, Alignment, ffi,\
    EllipsizeMode
```

Once applied, I'm happy to merge and prepare a quick release! 😄 

I hope to reply to your other questions soon.